app/controllers/api/base_controller/parameters.rb

```diff
@@ -100,6 +102,8 @@ def parse_filter(filter, operators)
                         end
                       end
 
+        filter_value = from_cid(filter_value) if filter_attr =~ /[_]?id$/ && cid?(filter_value)
```

Comment:
Meh, it's only used here for now, up to you.  it will eventually be needed by https://www.pivotaltracker.com/story/show/121850107